variants/XMC1100/config/XMC1100_Boot_Kit/pins_arduino.h

```diff
@@ -88,21 +92,23 @@ extern uint8_t SCK;
 #define A6   6
 #define A7   7
 
-#define AD_AUX_1    24  // AD_AUX
```

Comment:
So we keep these pinout assignment as it was for the 1.x and then these changes are done for 2.x or?
For the xmc1400 as mentioned, at it did not make it to the master, we can consider those changes ;)